### PR TITLE
Email after provisioning

### PR DESCRIPTION
**Pull request by Kolezhanchik**
edit and create product emails are sent after provisioning

### CODE REVIEW

**Comment by okanji** [comment]
Thanks @Kolezhanchik, please see the review

File: src/resolvers/mutations/privateCloudRequestDecision.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ const privateCloudRequestDecision: MutationResolvers = async (
     }
   }
 
-  sendMakeDecisionEmails(request);
+  // sendMakeDecisionEmails(request);
```
[the remark this answers is not in the document]
Comment (okanji) [reply src/resolvers/mutations/privateCloudRequestDecision.ts:74]: @Kolezhanchik

File: src/controllers/provisionerCallbackHandler.ts [hunk 2]
```diff
@@ -48,8 +50,33 @@ const provisionerCallbackHandler = async (req, res) => {
...
+        },
+        requestedProject: {
+          include: {
+            projectOwner: true,
+            primaryTechnicalLead: true,
+            secondaryTechnicalLead: true,
+          },
+        },
+      },
+    }).then((res) => {
```
[the remark this answers is not in the document]
Comment (Kolezhanchik) [reply src/controllers/provisionerCallbackHandler.ts]: done

File: src/resolvers/mutations/privateCloudRequestDecision.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kolezhanchik) [reply src/resolvers/mutations/privateCloudRequestDecision.ts]: done

File: src/controllers/provisionerCallbackHandler.ts [hunk 3]
```diff
@@ -48,6 +50,31 @@ const provisionerCallbackHandler = async (req, res) => {
...
+          include: {
+            projectOwner: true,
+            primaryTechnicalLead: true,
+            secondaryTechnicalLead: true,
+          },
+        },
+      },
+    })
+
+    sendMakeDecisionEmails(emailRequest)
```
[the remark this answers is not in the document]
Comment (Kolezhanchik) [reply src/controllers/provisionerCallbackHandler.ts]: done

File: src/controllers/provisionerCallbackHandler.ts [hunk 4]
```diff
@@ -48,6 +50,31 @@ const provisionerCallbackHandler = async (req, res) => {
       create: requestedProject
     });
 
+    const emailRequest = await prisma.privateCloudRequest.findUnique({
```
[the remark this answers is not in the document]
Comment (Kolezhanchik) [reply src/controllers/provisionerCallbackHandler.ts]: done

**Comment by okanji** [comment]
Great @Kolezhanchik! 💯

**Pull request merged**